<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyPannel</class>
 <widget class="QWidget" name="PropertyPannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>信息</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/30.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>W：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>H:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_widget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_height">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/31.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_X_up">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y_up">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/2x_eyedroper.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_R">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_G">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_B">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#0055ff;&quot;&gt;+&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_X_down">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y_down">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>转换</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton_translation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>平移   dx：</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_scale">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>缩放    ↔</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rotate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>旋转    ∠</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dx">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dy:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sx">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>95</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;↕&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sy">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dy">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dthea">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_relation">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>关联</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_apply">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_t">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tr">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_bl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_b">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_br">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_l">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_c">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_r">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>描绘</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>22</height>
      </size>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icon/line1.png</normaloff>:/icon/line1.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icon/line2.png</normaloff>:/icon/line2.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icon/line3.png</normaloff>:/icon/line3.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icon/line4.png</normaloff>:/icon/line4.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icon/line5.png</normaloff>:/icon/line5.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icon/line6.png</normaloff>:/icon/line6.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>线型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>线宽：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_linewidth">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>颜色:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_getpencolor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_showline">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>填充</string>
    </attribute>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风格：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>没有</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>默认填充</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充93.75%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充87.50%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充62.50%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充50.00%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充37.50%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充12.50%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密度填充06.25%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>横线填充</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>竖线填充</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>网格填充</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>正斜线填充</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>反斜线填充</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>斜网格填充</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>颜色：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_getbrushcorol">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_showBrush">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
